xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxHorizontalLine.de.xml

```diff
@@ -47,8 +47,8 @@
 
 |=Funktion|=XWiki Syntax 1.0|=Ergebnis
 |Einfache horizontale Linie|{{{----}}}|(((----)))
-|Parametrisierte horizontale Linie|&lt;hr style="color:blue"/&gt;|(((
-(% style="color:blue" %)
+|Parametrisierte horizontale Linie|&lt;hr style="border-color:blue"/&gt;|(((
```

Comment:
Those changes do not concern the translation and should not be merged.

Reply:
I'm actually wrong here: @tmortagne  edited the base file as part of https://github.com/xwiki/xwiki-platform/commit/5afed0ae1fc525c5e332951a61c23230ee9788b7 changing this. Would have been probably better to edit all translations files too, to avoid this kind of diff in the translation PR. So I'll merge this PR.